railties/test/console_helpers.rb

```diff
@@ -18,8 +18,8 @@ def assert_output(expected, io, timeout = 10)
 
     assert_includes output, expected, "#{expected.inspect} expected, but got:\n\n#{output}"
   end
+end
 
-  def available_pty?
-    defined?(PTY) && PTY.respond_to?(:open)
-  end
+def available_pty?
```

Comment:
To make available_pty? method available outside of test methods, moving it outside of ConsoleHelpers module as other helper method does.